docs/package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "next",
     "start": "next start",
     "build": "next build ",
-    "lint": "eslint . --max-warnings 40",
+    "lint": "eslint . --max-warnings 0",
```

Comment:
LFG

Reply:
Let's re-enable lint check in the next.config.json as well!

Reply:
After chatting, we realized we don't need it. A GitHub Action does the check already.